.github/workflows/gtest-bare-metal.yml

```diff
@@ -59,13 +66,15 @@ jobs:
 
       - name: Checkout MTL
         uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+        with:
+          ref: '${{ inputs.branch-to-checkout || github.head_ref || github.ref }}'
 
       - name: Checkout DPDK
         uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
         with:
           repository: 'DPDK/dpdk'
-          ref: v${{  env.DPDK_VERSION  }}
-          path: dpdk
+          ref: 'v${{  env.DPDK_VERSION  }}'
```

Comment:
how about   DPDK_VERSION: 'v23.11'
then simply ref: '${{ env.DPDK_VERSION }}' ?

Reply:
Can't be done as we are referring the dpdk patch directory later on using dpdk version